src/main/features/claimant-response/draft/draftClaimantResponse.ts

```diff
@@ -22,12 +23,13 @@ export class DraftClaimantResponse extends DraftDocument {
   alternatePaymentMethod?: DraftPaymentIntention
   courtCalculatedPaymentIntention?: PaymentIntention
   courtOfferedPaymentIntention?: PaymentIntention
-  courtDecisionType?: DecisionType
+  decisionType?: DecisionType
   freeMediation?: FreeMediation
   paidAmount?: PaidAmount
   rejectionReason?: RejectionReason
```

Comment:
`rejectionReason` remains here 👍

Reply:
as discussed, this remains here for now but will be moved to courtDetermination as part of refactoring story